scraper.rb

```diff
@@ -1,38 +1,67 @@
 require 'scraperwiki'
 require 'mechanize'
 
-starting_url = 'http://www.sorell.tas.gov.au/publications/currently-advertised-applications/'
+def scrape()
```

Comment:
You're mixing up refactoring with some changes which are not at all clear from the commit message. So, this whole commit is difficult for me to review